src/common/components/community-cover/index.tsx

```diff
@@ -163,9 +163,13 @@ export class CommunityCover extends Component<Props> {
 
     let bgImage = global.theme === "day" ? coverFallbackDay : coverFallbackNight;
     if (community) {
-      bgImage = `https://images.ecency.com/${global.canUseWebp ? "webp/" : ""}u/${
-        community.name
-      }/cover`;
+      if (Object.keys(account).includes("profile") && account["profile"].cover_image) {
```

Comment:
@MTSaeed This is unnecesary, because our image server already does the checking for profile.cover_image. Please keep `.../cover`.